.gitmodules

```diff
@@ -1,6 +1,7 @@
 [submodule "third_party/abseil-cpp"]
 	path = third_party/abseil-cpp
 	url = https://github.com/abseil/abseil-cpp.git
+	branch = lts_2020_09_23
```

Comment:
I don't think we should add the branch here. I think that basically all this controls is what happens when someone tries to update with something like `git submodule update --remote`. If we had something automatically updating then this would be useful to keep it on a branch that only gets bug fix patches, but I suspect that the next time we update we won't want to be staying on the 2020_09_23 release and nothing is automatically updating this.

Reply:
Okay, the change for this file is reverted.